ts/features/bonus/siciliaVola/components/CheckResidenceComponent.tsx

```diff
@@ -30,16 +46,16 @@ const CheckResidenceComponent = (props: Props): React.ReactElement => {
     primary: false,
     bordered: true,
     onPress: props.back,
-    title: "Back"
+    title: I18n.t("global.buttons.back")
```

Comment:
```suggestion
    onPress: props.cancel,
    title: I18n.t("global.buttons.cancel")
```
According to the mockups the button should show undo instead of back.
<img src="https://user-images.githubusercontent.com/11773070/128831156-97f9cb87-5c4a-48af-834c-e2289d5951de.png" width=300>
The action to dispatch would be:
`cancel: () => dispatch(svGenerateVoucherCancel())`

Reply:
Good catch! Fixed https://github.com/pagopa/io-app/pull/3288/commits/a4630ff89e8c44f21ebe2a43bd1c9b0f82bd6534